functions.php

```diff
@@ -468,14 +468,15 @@ function interconnection_modify_polylang_query( $query ) {
 function interconnection_remove_homepage_sticky_posts( $query ) {
 	// Get the last added sticky post - last in array.
 	$sticky            = get_option( 'sticky_posts' );
-	$exclude_from_grid = $sticky ? array( $sticky[ count( $sticky ) - 1 ] ) : '';
+	$exclude_from_grid = is_array( $sticky ) ? array_pop( $sticky ) : '';
```

Comment:
What happens if `$sticky` is empty?

`array_pop` mutates the array, whereas reading from the index did not. There's no reference in play so this should be fine, but wanted to note that to be thorough.

Reply:
If you pop from an empty array, you'll get `null` but without throwing a notice, which is what I wanted:

```
wp> $sticky = [];
=> array(0) {
}
wp> array_pop( $sticky );
=> NULL
```
```
wp> $sticky = [];
=> array(0) {
}
wp> array( $sticky[ count( $sticky ) - 1 ] )
Notice: Undefined offset: -1 in phar:///usr/local/bin/wp/vendor/wp-cli/shell-command/src/WP_CLI/Shell/REPL.php(46) : eval()'d code on line 1
=> array(1) {
  [0]=>
  NULL
}
```